paper_io_sim.tex

```diff
@@ -769,10 +776,14 @@
         simulated cache content exactly matches reality, since all files fit in page cache. With 100~GB files, a slight
         discrepancy is observed after Write 2, which explains the
         simulation error previously mentioned in Read 3. In the real execution indeed,
-        File 3 is entirely cached after Write 2, whereas in the simulated execution, part
-        of it is on disk. This is due to the fact that the simulated flushing and eviction
-        was triggered after File 3 was already written, while in reality these operations
-        happened before File 3 was written \tristan{Dzung, pls check that. Also, why doesn't it happen irl?}.
+        File 3 is entirely cached after Write 2, whereas in the simulated execution, 
+        only a part of it is cached. 
+        This is due to the fact that the in simulation, data is flushed and evicted 
+        from the inactive list first, then from the active list. Thus, File 2, which is 
+        accessed again by Read 2 and is cached in the active list, is not flushed and evicted, 
+        while File 3 is flushed and evicted since it is accessed once only and cached 
+        in the inactive list. This may not correctly simulate what really happens 
+        in reality. 
```

Comment:
"this may not correctly simulate what really happens in reality" is too vague, can you be more specific?

Reply:
This flushing and eviction order in simulation may not correctly reflect the order in reality.